### PR TITLE
JIRA: AOAF-424

### PR DESCRIPTION
**Pull request by ch-skathirvel**
Description: Added language parameter in the redirect url.

### CODE REVIEW

**Comment by okoye-companieshouse** [comment]
@ch-skathirvel can you add a unit test to demonstrate that the url contains the lang query? Just need to modify line 23 of https://github.com/companieshouse/presenter-account-web/blob/main/test/routers/home.unit.ts to check for the lang query as well

File: src/middleware/authentication.middleware.ts [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 import { authMiddleware, AuthOptions } from "@companieshouse/web-security-node";
 import { env } from "../config";
-import { removeLangFromUrl } from "../utils/query";
 import { Urls, PrefixedUrls } from "../constants";
 export const authenticationMiddleware = (req: Request, res: Response, next: NextFunction) => {
     const CHS_URL = env.CHS_URL;
-    const returnUrl = getReturnCallbackURL(req);
     const authMiddlewareConfig: AuthOptions = {
         chsWebUrl: CHS_URL,
-        returnUrl: removeLangFromUrl(returnUrl)
```
[the remark this answers is not in the document]
Comment (ch-skathirvel) [reply src/middleware/authentication.middleware.ts:11]: The confirmation URL doesn't send the query string at the moment, so the return call back url logic works fine now. Do you still prefer to change it as part of this story?

**Review by okoye-companieshouse — APPROVED** [review]
LFBM

File: src/middleware/authentication.middleware.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ch-skathirvel) [reply src/middleware/authentication.middleware.ts:11]: As discussed, path doesn't work, hence updated with base url. Please review now.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**